crates/forge/src/main.rs

```diff
@@ -68,12 +79,10 @@ fn main_execution() -> Result<()> {
         .output()
         .context("Failed to build contracts with Scarb")?;
     if !build_output.status.success() {
-        bail!(
-            "Scarb build didn't succeed:\n\n{}",
-            String::from_utf8_lossy(&build_output.stdout)
```

Comment:
Why is this removed?

Reply:
Nope, we redirect both stdout and stderr, that's why I removed it